spec/support/normalize_indent.rb

```diff
@@ -2,8 +2,8 @@ module IndentNormalizer
   # Strips off excess leading indentation from each line so we can use Heredocs
   # for writing code without having the leading indentation count.
   def normalize_indent(code)
-    leading_indent = code[/^(\s*)/, 1]
-    code.lstrip.gsub(/\n#{leading_indent}/, "\n")
+    leading_indent = code[/^(\s*?)(\n|\S)/, 1]
+    code.gsub(/^#{leading_indent}/, '')
```

Comment:
It was not working as really intended, and got an issue with first blank line. Seems like I was not able to test warning when first line is blank, because it was stripped.